apis/python/test/common.py

```diff
@@ -183,3 +185,8 @@ def accuracy(result, gt, external_ids_offset=0, updated_ids=None):
         total += len(temp_result)
         found += len(np.intersect1d(temp_result, gt[i]))
     return found / total
+
+
+def random_name(name: str) -> str:
```

Comment:
Please add a comment about why we use this.

Reply:
Added